python/eos/analysis.py

```diff
@@ -81,11 +83,18 @@ def __init__(self, priors, likelihood, global_options={}, manual_constraints={})
         eos.debug('manual_constraints:')
         for cn, ce in manual_constraints.items():
             eos.debug(' - {name}'.format(name=cn))
+        eos.debug('fixed_parameters:')
+        for cn, ce in fixed_parameters.items():
```

Comment:
Should that not me ``pn, pv`` as in parameter name, parameter value?

Reply:
It definitely should :)